basil/utils/USBBinds.py

```diff
@@ -0,0 +1,245 @@
+import os
+
+from pathlib import Path
+
+from six import string_types
+
+import pyvisa
+
+import ruamel.yaml
+
+
+def queryIdentification(rm, resource, baud_rate, read_termination="\n", write_termination="\n", timeout=1000 * 5):
```

Comment:
Please use pythonic function names (snake case instead of camel case). Also: the default read/write termination defined in the Serial connection driver (https://github.com/SiLab-Bonn/basil/blob/master/basil/TL/Serial.py#L31-L32) is `None`